lib/components/viewers/viewers.css

```diff
@@ -102,48 +102,86 @@
 
 /* stop viewer styles */
 
-.otp .stop-viewer .route-row {
-}
 .otp .stop-viewer .route-row-container,
 .related-panel-container {
-  border-radius: 10px;
-  box-shadow: 2px 2px 5px 1px rgba(0, 0, 0, 0.1);
   list-style-type: none;
   padding-left: 0;
 }
+.otp .stop-viewer .departures {
+  box-shadow: 2px 2px 5px 1px rgb(0 0 0 / 10%);
+}
+
+.otp .stop-viewer .day-container .route-row-container li .header.stop-view {
+  border-radius: 0;
+}
+
+.otp .stop-viewer .day-container p {
+  font-weight: 500;
+  margin: 15px 10px;
+}
+
+.otp
+  .stop-viewer
+  .day-container:first-of-type
+  .route-row-container
+  li:first-of-type
+  .header.stop-view {
+  border-top-left-radius: 10px;
+  border-top-right-radius: 10px;
+}
+.otp
+  .stop-viewer
+  .day-container:last-of-type
+  .route-row-container
+  li:last-of-type
+  .header.stop-view {
+  border-bottom-left-radius: 10px;
+  border-bottom-right-radius: 10px;
+}
+
 .otp .stop-viewer .route-row:first-of-type {
   margin-top: 10px;
 }
 
 .otp .stop-viewer .route-row .header {
   align-items: center;
-  display: grid;
-  grid-template-columns: 2fr 1fr;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  overflow: hidden;
   width: 100%;
 }
 .otp .stop-viewer .route-row .header .route-name {
   align-items: center;
-  display: grid;
-  font-size: 42px;
-  gap: 5px;
-  grid-template-columns: 4ch 2fr;
+  display: flex;
+  gap: 10px;
 }
 .otp .stop-viewer .route-row .header .route-name span {
+  display: -webkit-box;
   font-size: 15px;
+  line-clamp: 3;
+  overflow: hidden;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 3;
```

Comment:
I'm not sure we should use webkit specific rules. We have a post processor that's supposed to automatically apply these if we use the generic rules! What is this doing anyway? When we originally built this we moved away from `line-clamp` because it was so poorly supported